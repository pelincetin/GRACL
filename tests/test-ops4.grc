void printb(bool b){
  if (b){
    printi(1);
  }
  else {
    printi(0);
  }
}
int main()
{
  printb(1 >= 2);
  printb(1 >= 1);
  printb(2 >= 1);
  printi(99);

  printb(4.0 >= 1.0);
  printb(1.0 >= 4.0); 
  printb(2. >= -1.0); 
  printb(1 >= -1); 
  printb(1.0 >= 1.0);
  printb(4.0 >= -4.0);   
  printb(0.00 >= -0.);
  printb(4 >= -0); 
  printb(-4 >= -0); 
  printb(-1.0 >= -0.00000);       

  return 0;
}
