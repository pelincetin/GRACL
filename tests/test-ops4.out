0
1
1
99
1
0
1
1
1
1
1
1
0
0

